packages/react-utilities/src/hooks/useControllableState.ts

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { useConst } from './useConst';
+
+type NonUndefined<T> = T extends undefined ? never : T;
+
+type UseAutoControlledOptions<ControllableState, State> = {
```

Comment:
_nit_ Can we export this type?

Reply:
I just don't know why, their existence is just for internal use ?